backend/apps/tenant_config_app.py

```diff
@@ -94,18 +69,18 @@ def update_knowledge_list(
             tenant_id=tenant_id, user_id=user_id, index_name_list=knowledge_list)
         if result:
             return JSONResponse(
-                status_code=200,
+                status_code=HTTPStatus.OK,
                 content={"message": "update success", "status": "success"}
             )
         else:
             return JSONResponse(
```

Comment:
建议异常均使用HTTPException抛出，由base_app exception_handler 做统一转化
```suggestion
            raise HTTPException(
```